feat(curriculum): add pin-extractor workshop to FS (#60925)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-pin-extractor/68531fbb934d735415053274.md

---
id: 68531fbb934d735415053274
title: Step 4
challengeType: 20
dashedName: step-4
---

# --description--

Now that you have added a function call you can see any error created inside the function in the terminal, and you can always use `print` to see the value of things.

> There is a digit of the pin hidden in any line, so inside the function use `split` to divide the string in a list of lines (you can split on the newline character `\n` for that), and assign the resulting list to a variable called `lines`.

Reviewer: ```suggestion
There is a digit of the pin hidden in any line, so inside the function use the `split` method to divide the string in a list of lines. Split the lines on the newline character (`\n`), and assign the resulting list to a variable called `lines`.
```
